src/components/MaterialUI/Navigation/TabBar.js

```diff
@@ -116,17 +120,17 @@ const MuiBar = ({ module, moduleName, pages }) => {
 	const modifiedTabs = useSelector(getModifiedTabs(pagesParams));
 	const tabsWithErrors = useSelector(getTabsWithErrors(pagesParams));
 
+	const tabLabels = [];
+
 	const handleChange = (_, value) => {
-		if (value === false) {
+		if (!!!value) {
```

Comment:
Possible de changer ça pour une condition plus claire, genre:

value < pages.lenght 

?

Reply:
c'est seulement si c'est false ou 0, les deux valeurs sont possibles, 
je peux faire (value === false || value === 0) mais je trouve ça plus compact de l'autre façon

Reply:
Je comprends, ceci dit, les double-bang et les triple-bang ne sont pas souvent explicites.

De plus getNewIndex retourne toujours un numérique, donc ceci sera suffisant et ça me dit exactment ce qu'on veut véfifier. C'est comme tu veux.

value > 0 && value < pages.lenght

Reply:
so I will need to inverse the ordering of the conditionals to have
(value > 0 && value < pages.length) to push the actual href else, we go to the module page.